src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball firstBall = lottery.getRandomBall();
```

Comment:
This method should create a ball with a random color and a random number (the maximum possible number should be 100) and return this ball from the method.
What in Java can we use to iterate a part of program several times and avoid creating Ball line by line?

Reply:
Hint: https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-for-loop-for-creating-several-objects-of-the-same-class